api/v1beta2/bgppeer_types.go

```diff
@@ -55,6 +59,11 @@ type BGPPeerSpec struct {
 	// +optional
 	KeepaliveTime metav1.Duration `json:"keepaliveTime,omitempty"`
 
+	// Requested BGP connect time, controls how long BGP waits between connection attempts to a neighbor.
+	// +kubebuilder:default:="120s"
+	// +optional
+	ConnectTime metav1.Duration `json:"connectTime,omitempty"`
```

Comment:
why does it matter if it will land in all the frr configurations?
i don't really understand why this is a bad thing.

i do think that its a better user experience to have the default values as part of the documentation and well known, instead of making the users search for it by themselves in frr docs.  Also if we already support a feature i think its better to have full control over the value, and not be deepened by frr, i think if frr do change the value one day it can change the value for the users without notice. i think its better to  have full control for the value.

how ever i do think using a pointer can give  a neat implementation to check easily if the value is set for native mode.

Reply:
I understand the general sentiment, and I wrote that too (when I mentioned that it's on the defensive side), and that's why I asked for @oribon 's view. We have good and comprehensive CI that should guarantee things will work.

> why does it matter if it will land in all the frr configurations? i don't really understand why this is a bad thing.
> 

For example, we hit rendering problems with default values: 
https://github.com/FRRouting/frr/issues/14242


> i do think that its a better user experience to have the default values as part of the documentation and well known, instead of making the users search for it by themselves in frr docs. Also if we already support a feature i think its better to have full control over the value, and not be deepened by frr, i think if frr do change the value one day it can change the value for the users without notice. i think its better to have full control for the value.
> 

I totally agree with this part, I am just afraid of side effects.

> how ever i do think using a pointer can give a neat implementation to check easily if the value is set for native mode.

Let's wait a bit before moving on, I am open to both alternatives

Reply:
talked with @fedepaol , I think going forward any field that is FRR specific should be pointerized (and omitted from frr.conf) like we do with bfdprofiles, the main reason being them understanding these concepts better than us and at this point we're frr consumers, so it feels natural to defer it to whatever they do and generate a simpler configuration (although I agree we do lose some "control/visibility"). 
about the documentation part I think writing what frr assigns as default is enough as I believe a user who wouldn't set this field doesn't really care whether we set the default or not.

Reply:
Tanks @oribon and @fedepaol for the great explanations! its always nice to understand why this is the chosen approach :)   changed connect time to be pointerized